chicagoland.pandemicresponsecommons.org/manifest.json

```diff
@@ -47,6 +47,7 @@
     "tier_access_level": "regular",
     "tier_access_limit": "5",
     "useryaml_s3path": "s3://cdis-gen3-users/covid19/user.yaml",
+    "covid19_etl_bucket": "https://opendata.datacommons.io/",
```

Comment:
I don't think this is correct. the current value is this: 
```            - name: S3_BUCKET
              value: s3://opencdn-databucket-gen3
```

@paulineribeyre Can you confirm the correct bucket name here?

Reply:
ah shoot, yeah, the bucket name is `s3://opencdn-databucket-gen3` and the CDN URL is `https://opendata.datacommons.io`

@atharvar28, I forgot the portal uses the URL, not the bucket name, so we won't be able to pull that value from the manifest for the portal. But that's ok, the portal can keep pulling from its own config, and the ETLs can pull from the manifest

Reply:
I have updated the s3 url. Thank you guys 👍